src/templates/static/about/contact.jsx

```diff
@@ -6,69 +6,98 @@ const support_email = 'support@binary.com';
+                                                    { text: it.L('Russia'),         value: it.L('[_1] (Toll Free)', '8 10 8002 8553011') },
+                                                    { text: it.L('United Kingdom'), value: it.L('[_1] (Toll Free)', '+44 (0) 1666 800042, 0800 011 9847'), selected: true },
+                                                ]}
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className='gr-row'>
+                                        <div className='gr-12' id='display_cs_telephone'>
+                                            {('+44 (0) 1666 800042')}
+                                            <br />
+                                            {it.L('[_1] (Toll Free)', '0800 011 9847')}
+                                        </div>
+                                    </div>
                                 </div>
                             </div>
-                            <span className='gr-8 gr-prefix-4 gr-12-t gr-12-p gr-padding-10 gr-12-m' id='display_cs_telephone'>
-                                {('+44 (0) 1666 800042')}
-                                <br />
-                                {it.L('[_1] (Toll Free)', '0800 011 9847')}
-                            </span>
                         </div>
-                        <p className='ja-hide'>{it.L('If you are not located in the above-mentioned countries, simply dial any of our contact numbers for help.')}</p>
-                        <div className='gr-padding-10 hint calls_recording'>
+                        <div className='gr-row'>
+                            <div className='gr-11 gr-10-t gr-12-p gr-12-m'>
+                                <p className='ja-hide'>{it.L('If you are not located in the above-mentioned countries, simply dial any of our contact numbers for help.')}</p>
+                            </div>
+                        </div>
+                        <div className='hint calls_recording'>
                             * {it.L('All calls are recorded for training and monitoring purposes')}
                         </div>
+                        <div className='gr-parent ja-hide'>
+                            <div className='chat'>
+                                <div className='chat-container'>
+                                    <img className='chat-icon' src={it.url_for('images/pages/contact/chat-icon.svg')} />
+                                    <a
+                                        className='chat-link'
+                                        href='https://binary.desk.com/customer/widget/chats/new'
+                                        target='_blank'
+                                        rel='noopener noreferrer'
+                                    >
```

Comment:
you can just use a secondary button style https://style.binary.com/#button-section, then no need to add so much css for border, color, etc